src/request-api.js

```diff
@@ -84,7 +85,7 @@ function requestAPI (config, path, args, qs, files, buffer, cb) {
   const port = config.port ? `:${config.port}` : ''
 
   const opts = {
-    method: files ? 'POST' : 'GET',
+    method: 'POST',
```

Comment:
Let's check with @whyrusleeping, I've changed this after a conversation with him when we were debugging some npm-on-ipfs stuff.